music/src/core/sequences.ts

```diff
@@ -656,15 +656,16 @@ function trimHelper(
           (truncateEndNotes || n[endKey] <= end));
 
   // Shift the sequence to start at 0.
+  result[totalKey] -= start;
   for (let i = 0; i < result.notes.length; i++) {
     result.notes[i][startKey] -= start;
     result.notes[i][endKey] -= start;
 
     if (truncateEndNotes) {
-      result.notes[i][endKey] = Math.min(result.notes[i][endKey], end);
+      result.notes[i][endKey] = Math.min(result.notes[i][endKey],
```

Comment:
Yeah, i think you're right, and this makes it consistent with https://github.com/tensorflow/magenta/blob/master/magenta/music/sequences_lib.py#L124

Do you need to update .totalTime too? as per https://github.com/tensorflow/magenta/blob/master/magenta/music/sequences_lib.py#L126

Reply:
Should be already updated above (lines 653 and then 659). (I'm shifting it before, so that I can use it on the line below.)